### ISSUE TITLE
Inconsistent indention for EQ_SUB

### ISSUE THREAD

**Issue by martin-mfg** [r-lib/styler#351]
The following code was styled using the latest development version of styler:

```R
a_very_long_function_name <- function(parameter_name =
                                      "a very long default value") {
  NULL
}

a_very_long_function_name(
  parameter_name =
    "a very long default value"
)
```

What confuses me here is that in the function call, `"a very long default value"` is indented by 1 more level than `parameter_name =`, but in the function definition this is not the case.
I think there is no rule in the tidyverse style guide that governs this behaviour, so in my opinion it would be reasonable to apply the same indention strategy in both cases.

**Comment by lorenzwalthert** [comment]
Thanks @martin-mfg, this is indeed unexpected. I think I know where it stems from and I will look into it as soon as possible.

**Comment by lorenzwalthert** [comment]
Blocked by #352.

**Comment by lorenzwalthert** [comment]
Blocked by #357.

**Comment by lorenzwalthert** [comment]
Solved in #353 but this is blocked by #351.

**Comment by lorenzwalthert** [comment]
Closed with #353.
